### PR TITLE
Development

### PR DESCRIPTION
**Pull request by BalintMendli**
Please review.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,1388 @@
...
+  display: flex;
+  align-items: center;
+}
+
+.nav-countdown-conf-ams17-timer-count {
+  font-size: 22px;
+}
+
+.nav-countdown-conf-ams17-item {
+  margin-left: 2px;
```
Comment (JamesOkunlade) [inline assets/css/style.css]: use margin shorthand. margin: 0 2px;

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,1388 @@
...
+  font-weight: normal;
+}
+
+.deals-header-more:not(:first-of-type) {
+  margin-right: 23px;
+}
+
+.deals-header-more:after {
+  content: '|';
+  margin-left: 10px;
```
Comment (JamesOkunlade) [inline assets/css/style.css]: Use margin shorthand. margin: 0 10px;

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,1388 @@
...
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.icon {
+  display: block;
+  background: transparent
+    url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/icons-sprite.svg?1520429355783)
```
Comment (JamesOkunlade) [inline assets/css/style.css]: Indent properly.

**Review by JamesOkunlade — APPROVED** [review]
### Superb work you did on this project!

Refactor according to my different comments on different lines in your CSS. When you're done, merge development to master, close pull request and fill the completion form.

Here are your thumbs ups :+1::+1:

Good luck with your future project.

P.S This project is a collaborative and I see all commits came from a single person. You might want to talk to Lydia why you worked on this project alone.

**Comment by BalintMendli** [comment]
Made the suggested changes. Thank you for your review.

**Pull request merged**